prow/knative/plugins.yaml

```diff
@@ -25,23 +26,8 @@ approve:
 # Settings for the "lgtm" plugin.
 lgtm:
 - repos:
-  # For now only start with knative-sandbox/net-contour
-  - knative-sandbox/.github
-  - knative-sandbox/downstream-test-go
-  - knative-sandbox/integration
-  - knative-sandbox/knobots
-  - knative-sandbox/kperf
-  - knative-sandbox/net-certmanager
-  - knative-sandbox/net-contour
-  - knative-sandbox/net-http01
-  - knative-sandbox/net-ingressv2
-  - knative-sandbox/net-istio
-  - knative-sandbox/net-kourier
-  - knative-sandbox/sample-controller
-  - knative-sandbox/sample-source
-  - knative-sandbox/reconciler-test
+   - "knative-sandbox"
```

Comment:
I am assuming `repos` is correct mostly based on the commented lines below.  @chizhg @chaodaiG any sanity checking would be wonderful 🙏

Reply:
I believe it's correct

Reply:
There are a few other places that seem to use `repos:` this way, so I figured it might be, but other places have `orgs:` too 😅

Reply:
You are definitely right about this. Generally prow configs are pretty consistent among each other, but when it comes to each individual plugins, there is no such good cohesion, as each plugin gets to decide the structure for itself, as well as how to interpret them (https://github.com/kubernetes/test-infra/blob/master/prow/plugins/config.go). This imo had become a tech debt and cleaning up is not an easy task.